change/@fluentui-react-conformance-9e218894-46ad-4650-a8c1-7bf66cf08a72.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

After the change I'm suggesting below, this should be patch
```suggestion
  "type": "patch",
```